src/frame/js/components/ChatInput.jsx

```diff
@@ -132,10 +132,10 @@ export class ChatInputComponent extends Component {
     }
 }
 
-export default connect(({appState, app, ui, conversation: {unreadCount}}) => {
+export default connect(({ui, conversation: {unreadCount}, config}) => {
     return {
-        imageUploadEnabled: appState.imageUploadEnabled,
-        accentColor: app.settings.web.accentColor,
+        imageUploadEnabled: config.imageUploadEnabled,
```

Comment:
config is a mix of what the `/config` endpoint return and local config (it's set here https://github.com/smooch/smooch-js/pull/572/files#diff-8f16faf8c766689cc06ac2f62eb9db67R138)